ArjunaCore/arjuna/classes/com/arjuna/ats/arjuna/logging/arjunaI18NLogger.java

```diff
@@ -1645,6 +1645,11 @@ public void warn_objectstore_JDBCImple_over_max_image_size(int imageSize,
 	@LogMessage(level = WARN)
 	void unexpected_exception(@Cause() Throwable arg0);
 
+	// TODO Delete this as soon as JBTM-3983 has been implemented
+	@Message(id = 12414, value = "Only AtomicActionRecoveryModule is supported in this phase of the implementation", format = MESSAGE_FORMAT)
```

Comment:
Perhaps the wording could be expanded a little as it is a bit general and could be a bit misleading in isolation in a log file without context?

I would suggest: "A recovery module other than com.arjuna.ats.internal.arjuna.recovery.AtomicActionRecoveryModule has been configured to be used by the recovery manager. Please note that if you issue a request where you expect the Recovery Manager to be blocked during suspension until all transactions are recovered, then only AtomicActionRecoveryModule is currently supported for this purpose. Please see https://issues.redhat.com/browse/JBTM-3893 for more details"

Reply:
Code has been modified. Thanks Tom

Reply:
Thank you